tasks/Task.cpp

```diff
@@ -86,8 +86,9 @@ void Task::updateHook()
     _ping_status.write(ping);
     if (ping.flag == 1) {
         auto rbs = convertToRBS(ping);
-        _local_pose.write(rbs);
+        _pose.write(rbs);
         auto attitude = convertToQuaterniond(ping);
+        _local_attitude.write(attitude);
```

Comment:
Why not just pass the orientation to the local pose? It actually describes the local pose frame